lib/db/models/SwapDeal.ts

```diff
@@ -0,0 +1,33 @@
+import Sequelize from 'sequelize';
+import { db } from '../../types';
+
+export default (sequelize: Sequelize.Sequelize, DataTypes: Sequelize.DataTypes) => {
+  const attributes: db.SequelizeAttributes<db.SwapDealAttributes> = {
+    myRole: { type: DataTypes.TINYINT, allowNull: false },
+    state: { type: DataTypes.TINYINT, allowNull: false },
+    stateReason: { type: DataTypes.STRING, allowNull: false },
+    r_hash: { type: DataTypes.STRING, allowNull: false },
+    r_preimage: { type: DataTypes.STRING, allowNull: false },
```

Comment:
I know we haven't changed the names of these variables yet, but lets go with camelCase for these two so that we don't need a breaking db change later on.